### DISCUSSION TITLE
Grafana session information by port number

### DISCUSSION THREAD

**Discussion by rvdheij** [grafana/grafana#72491] [Dashboards, Panels & Visualizations]
I run multiple container instances of Grafana with different versions or experiments that should not impact others. The URL for these sessions differs only in the port number when both containers run on the same host (e.g. port 3000 and 3100). An attempt to open a second browser tab to another instance on the same host will cause the first tab to fail with authentication errors.

I assume that's because the session information gets messed up. Multiple tabs to different hosts works just fine. I could bypass the issue with extra fake host names in my /etc/hosts but that also interferes with the SSL certificates.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82488. Feel free to continue the discussion around this there. Thank you!
